### ISSUE TITLE
binary packaging errors

### ISSUE THREAD

**Issue by jakoch** [thombashi/sqlitebiter#65]
Hey Tsuyoshi!

The `sqlitebiter.exe`'s of v0.27.0 do not work. Seems the pkg_resource 'jsonschema' is missing.

I'm using executables from https://ci.appveyor.com/project/thombashi/sqlitebiter/builds/23301467

The  output for the Python-2.7 version is:
```
sqlitebiter.exe --version
Traceback (most recent call last):
  File "cli.py", line 10, in <module>
  File "c:\users\appveyor\appdata\local\temp\1\pip-install-khe1ut\pyinstaller\PyInstaller\loader\pyimod03_importers.py", line 395, in load_module
  File "sqlitebiter\sqlitebiter.py", line 19, in <module>
  File "c:\users\appveyor\appdata\local\temp\1\pip-install-khe1ut\pyinstaller\PyInstaller\loader\pyimod03_importers.py", line 395, in load_module
  File "site-packages\pytablereader\__init__.py", line 28, in <module>
  File "c:\users\appveyor\appdata\local\temp\1\pip-install-khe1ut\pyinstaller\PyInstaller\loader\pyimod03_importers.py", line 395, in load_module
  File "site-packages\pytablereader\json\core.py", line 22, in <module>
  File "c:\users\appveyor\appdata\local\temp\1\pip-install-khe1ut\pyinstaller\PyInstaller\loader\pyimod03_importers.py", line 395, in load_module
  File "site-packages\pytablereader\json\formatter.py", line 11, in <module>
  File "c:\users\appveyor\appdata\local\temp\1\pip-install-khe1ut\pyinstaller\PyInstaller\loader\pyimod03_importers.py", line 395, in load_module
  File "site-packages\jsonschema\__init__.py", line 33, in <module>
  File "site-packages\pkg_resources\__init__.py", line 472, in get_distribution
  File "site-packages\pkg_resources\__init__.py", line 344, in get_provider
  File "site-packages\pkg_resources\__init__.py", line 892, in require
  File "site-packages\pkg_resources\__init__.py", line 778, in resolve
pkg_resources.DistributionNotFound: The 'jsonschema' distribution was not found and is required by the application
[14112] Failed to execute script cli
```

The output for the Python-3.7 version is:
```
sqlitebiter.exe --version
Traceback (most recent call last):
  File "cli.py", line 10, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "c:\python37-x64\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "sqlitebiter\sqlitebiter.py", line 19, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "c:\python37-x64\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "site-packages\pytablereader\__init__.py", line 28, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "c:\python37-x64\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "site-packages\pytablereader\json\core.py", line 22, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "c:\python37-x64\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "site-packages\pytablereader\json\formatter.py", line 11, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "c:\python37-x64\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 627, in exec_module
  File "site-packages\jsonschema\__init__.py", line 33, in <module>
  File "site-packages\pkg_resources\__init__.py", line 479, in get_distribution
  File "site-packages\pkg_resources\__init__.py", line 355, in get_provider
  File "site-packages\pkg_resources\__init__.py", line 898, in require
  File "site-packages\pkg_resources\__init__.py", line 784, in resolve
pkg_resources.DistributionNotFound: The 'jsonschema' distribution was not found and is required by the application
[14896] Failed to execute script cli
```

I'm not sure, when this error was introduced. 
I switched back to v0.23.8, which works for me.

Please test the version output of the executable on Appveyor to avoid this (maybe `test_script: C:\projects\sqlitebiter\dist\sqlitebiter.exe --version` is enough; but i'm not sure, if this fails the build, when execution of the exe fails). 

Regards, Jens

**Comment by jakoch** [comment]
Same for `sqlitebiter` from `sqlitebiter_0.27.0_amd64.deb`:
```
Traceback (most recent call last):
  File "cli.py", line 10, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
  File "sqlitebiter/sqlitebiter.py", line 19, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
  File "site-packages/pytablereader/__init__.py", line 28, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
  File "site-packages/pytablereader/json/core.py", line 22, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
  File "site-packages/pytablereader/json/formatter.py", line 11, in <module>
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
  File "site-packages/jsonschema/__init__.py", line 33, in <module>
  File "site-packages/pkg_resources/__init__.py", line 481, in get_distribution
  File "site-packages/pkg_resources/__init__.py", line 357, in get_provider
  File "site-packages/pkg_resources/__init__.py", line 900, in require
  File "site-packages/pkg_resources/__init__.py", line 786, in resolve
pkg_resources.DistributionNotFound: The 'jsonschema' distribution was not found and is required by the application
[45] Failed to execute script cli
```

**Comment by thombashi** [comment]
Hi @jakoch,
Thank you for your report.

I had fixed executable binaries at `sqlitebiter 0.27.1`.
In addition, added simple test case (version check as you described) for each executable at build time.

Executable binaries corrupted since `0.24.0`.

**Comment by jakoch** [comment]
Fixed.

Thank you so much for this great tool!
❤️
